test/ops/conv2d.js

```diff
@@ -3076,4 +3076,70 @@ describe('test conv2d', function() {
     };
     testConv2d(input, filter, expected, options);
   });
+
+  it('conv2d input=1x1x5x5 dilations=2', function() {
+    const input = {
+      shape: [1, 1, 5, 5],
+      data: new Float32Array([
+        0,  1,  2,  3,  4,  5,  6,  7,  8,  9,  10, 11, 12,
+        13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24,
+      ]),
+    };
+    const filter = {
+      shape: [1, 1, 3, 3],
+      data: new Float32Array(9).fill(1),
+    };
+    const expected = {
+      shape: [1, 1, 1, 1],
+      data: [108],
+    };
+    const options = {
+      dilations: [2, 2],
+    };
+    testConv2d(input, filter, expected, options);
+  });
+
+  it('conv2d input=1x5x5x1 dilations=4 nhwc', function() {
+    const input = {
+      shape: [1, 5, 5, 1],
+      data: new Float32Array([
+        0,  1,  2,  3,  4,  5,  6,  7,  8,  9,  10, 11, 12,
+        13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24,
+      ]),
+    };
+    const filter = {
+      shape: [1, 1, 2, 2],
+      data: new Float32Array(4).fill(1),
+    };
+    const expected = {
+      shape: [1, 1, 1, 1],
+      data: [48],
+    };
+    const options = {
+      dilations: [4, 4],
+      inputLayout: 'nhwc',
+    };
+    testConv2d(input, filter, expected, options);
+  });
+
+  it('conv2d input=1x65x65x960 dilations=4 nhwc', function() {
+    const input = {
+      shape: [1, 65, 65, 960],
+      data: new Float32Array(4056000).fill(0),
```

Comment:
I think the all zero inputs are not good for test. Could you generate some non-zero input data?

Reply:
OK, I will update this test, thanks @huningxin.

Reply:
Done to update test. 
@huningxin Please take another look, thanks.